src/components/question/Question.tsx

```diff
@@ -17,31 +17,36 @@ import IconTitleDescription from '~/components/icon-title-description/IconTitleD
 import AppChip from '~/components/app-chip/AppChip'
 import DragHandle from '~/components/drag-handle/DragHandle'
 import { IconButton } from '~/design-system/components/icon-button/IconButton'
+import RadioButton from '~/design-system/components/radio-button/RadioButton'
 
 import {
   ColorEnum,
   Question as QuestionInterface,
+  QuestionTypesEnum,
   TableActionFunc
 } from '~/types'
 import { styles } from '~/components/question/Question.styles'
 import { spliceSx } from '~/utils/helper-functions'
+import { determineQuestionType } from '../question-editor/QuestionEditor.constants'
```

Comment:
```suggestion
import { determineQuestionType } from '~/components/question-editor/QuestionEditor.constants'
```
Use global imports, please.